tap_github/scraping.py

```diff
@@ -85,3 +87,67 @@ def _scrape_dependents(url: str, logger: logging.Logger) -> Iterable[Dict[str, A
             time.sleep(1)
         else:
             url = ""
+
+
+def parse_counter(
+    tag: Union[Tag, NavigableString, None], logger: logging.Logger
+) -> int:
+    if not tag:
+        return 0
+    try:
+        title = tag["title"]  # type: ignore
+        if isinstance(title, str):
+            title_string = cast(str, title)
+        else:
+            title_string = cast(str, title[0])
+        return int(title_string.strip())
+    except KeyError:
+        logger.debug(
+            f"Could not parse counter {tag}. Maybe the GitHub page format has changed?"
```

Comment:
The error handling here and below is not consistent, one just logs a debug message (likely to be silenced by a normal log level of `INFO`) the other raises an exception, although the likely underlying cause is similar (page format change). Is this on purpose?

Reply:
Yes, the two other values are optional and may not appear on the page. I'll make it clearer through comments

Reply:
I actually decided to raise an error in both cases since the case contributors or dependents "not found" is covered by tag = None